e2e/cypress/integration/files_and_attachments/upload_files_spec.js

```diff
@@ -149,26 +159,32 @@ describe('Upload Files', () => {
     });
 
     it('MM-T341 Download link on preview - Image file (non SVG)', () => {
-        ['small-image.png', 'image-file.jpg', 'image-file.bmp', 'image-file.gif', 'image-file.tiff'].forEach((image) => {
-            cy.get('#centerChannelFooter').find('#fileUploadInput').attachFile(image);
-            cy.get('.post-image').should('be.visible');
-            cy.postMessage(MESSAGES.SMALL);
-            cy.uiWaitUntilMessagePostedIncludes(MESSAGES.SMALL);
-            cy.getLastPostId().then((postId) => {
-                cy.get(`#post_${postId}`).within(() => {
+        ['bmp-image-file.bmp', 'png-image-file.png', 'jpg-image-file.jpg', 'gif-image-file.gif', 'tiff-image-file.tif'].
+            forEach((image) => {
+                cy.get('#centerChannelFooter').find('#fileUploadInput').attachFile(image);
+                cy.get('.post-image').should('be.visible');
+                cy.postMessage(MESSAGES.SMALL);
+                cy.uiWaitUntilMessagePostedIncludes(MESSAGES.SMALL);
+                cy.getLastPostId().then((postId) => {
+                    cy.get(`#post_${postId}`).within(() => {
                     // # Click the thumbnail of a non-SVG image attachment to open the previewer
-                    cy.get('div.image-loaded').find('img').click();
+                        cy.get('div.image-loaded').find('img').click();
+                    });
                 });
-            });
-            cy.get('.a11y__modal').should('be.visible').within(() => {
+                cy.get('.a11y__modal').should('be.visible').within(() => {
                 // # Click the "Download" link in the previewer
-                cy.findByText('Download').should('be.visible').parent().then((fileAttachment) => {
-                    //* Image should download
-                    verifyLinkHasDownloadProperties(fileAttachment, image);
```

Comment:
`verifyLinkHasDownloadProperties` was changed and improved in this PR, so made the change